aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.cpp

```diff
@@ -1385,7 +1389,7 @@ mha_fwd_kvcache(at::Tensor &q,                 // batch_size x seqlen_q x num_he
 
     auto round_multiple = [](int x, int m) { return (x + m - 1) / m * m; };
     const int head_size = round_multiple(head_size_og, 8);
-    const int head_size_rounded = round_multiple(head_size, 32);
+    const int head_size_rounded = round_multiple(head_size, 32) < 224 ? round_multiple(head_size, 32) : 256;
```

Comment:
why the change here?

Reply:
Upstream removed explicit instantiating of the 224 head-dim template so this rounds up to 256 (mirroring upstream)